package.json

```diff
@@ -38,19 +38,39 @@
 		"compile": "node ./node_modules/vscode/bin/compile -watch -p ./"
     },
 	"contributes": {
-		"keybindings": [
+		"keybindings": [			
 			{
 				"command": "PowerShell.OnlineHelp",
 				"key": "ctrl+f1",
 				"when": "editorTextFocus && editorLangId == 'powershell'"
 			},
 			{
+				"command": "PowerShell.ExpandAlias",
+				"key": "ctrl+f5",
+				"when": "editorTextFocus && editorLangId == 'powershell'"
+			},
+			{
 				"command": "PowerShell.RunSelection",
 				"key": "f8",
 				"when": "editorTextFocus && editorLangId == 'powershell'"
 			}
 		],
-		"commands": [
+		"commands": [			
+			{
+				"command": "PowerShell.ExpandAlias",
+				"title": "Expand Alias",
+				"category": "PowerShell"
+			},		
+			{
+				"command": "PowerShell.GetInstalledModule",
```

Comment:
Looks like these two commands got left in by accident.  Can you make another commit that removes them?